doc/doxygen/Doxyfile

```diff
@@ -272,7 +272,7 @@ EXPAND_ONLY_PREDEF     = YES
 SEARCH_INCLUDES        = YES
 INCLUDE_PATH           =
 INCLUDE_FILE_PATTERNS  =
-PREDEFINED             = __attribute__((x))=
+PREDEFINED             = __attribute__((x))=  DOXYGEN_RUN
```

Comment:
I ran doxygen a couple of times and never saw this being set. Beats me when it's supposed to be set on to what.